SQL/New_patches/2018-10-22-ExpireAllPasswords.sql

```diff
@@ -0,0 +1,6 @@
+-- This forces a password reset for all users the next time they login. Before
+-- this patch, all passwords changed by the front-end were stored in plaintext.
+UPDATE users SET Password_expiry='1990-01-01';
+-- Remove old plaintext passwords. Also affects old history entries when the
+-- column was named `hash` instead of `Password_hash`
+DELETE FROM history WHERE col LIKE "%hash%" AND tbl='users';
```

Comment:
I don't see why this uses `like '%hash%'` instead of `= 'password_hash'`

Reply:
IIRC I saw some instances of passwords (or their md5 hashes) being stored in the history table from when that column was named `md5hash` or whatever it was.

Reply:
Then use an OR to cover both cases.. `%hash%` wouldn't catch it anyways (IIRC the column was called `Password_md5` back when it was an md5)